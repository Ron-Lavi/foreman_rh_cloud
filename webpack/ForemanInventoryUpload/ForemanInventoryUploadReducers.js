import { combineReducers } from 'redux';
import { reducers as accountListReducers } from './Components/AccountList';
import { reducers as dashboardReducers } from './Components/Dashboard';
import { reducers as filterReducers } from './Components/InventoryFilter';

export default {
  inventoryUpload: combineReducers({
    ...accountListReducers,
    ...dashboardReducers,
    ...filterReducers,
  }),
};
