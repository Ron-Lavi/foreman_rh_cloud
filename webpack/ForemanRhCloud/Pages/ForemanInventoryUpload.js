import React from 'react';
import { Grid } from 'patternfly-react';
import PageLayout from 'foremanReact/routes/common/PageLayout/PageLayout';
import { translate as __ } from 'foremanReact/common/I18n';
import AccountList from '../Components/AccountList';
import PageHeader from '../Components/PageHeader';

const ForemanInventoryUpload = () => (
  <PageLayout header={__('Red Hat Inventory Upload')} searchable={false}>
    <Grid fluid className="inventory-upload">
      <PageHeader />
      <AccountList />
    </Grid>
  </PageLayout>
);

export default ForemanInventoryUpload;
