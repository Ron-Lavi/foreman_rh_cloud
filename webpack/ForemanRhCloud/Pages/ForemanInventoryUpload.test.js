import { testComponentSnapshotsWithFixtures } from '@theforeman/test';

import ForemanInventoryUpload from './ForemanInventoryUpload';

const fixtures = {
  'render without Props': {},
};

describe('ForemanInventoryUpload', () =>
  testComponentSnapshotsWithFixtures(ForemanInventoryUpload, fixtures));
