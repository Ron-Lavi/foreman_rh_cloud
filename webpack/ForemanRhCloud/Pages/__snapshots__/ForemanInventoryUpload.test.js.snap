// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForemanInventoryUpload render without Props 1`] = `
<PageLayout
  header="Red Hat Inventory Upload"
  searchable={false}
>
  <Grid
    bsClass="container"
    className="inventory-upload"
    componentClass="div"
    fluid={true}
  >
    <PageHeader />
    <Connect(AccountList) />
  </Grid>
</PageLayout>
`;
